services/jaeger/2.43.0/defaults/cm.yaml

```diff
@@ -6,6 +6,7 @@ metadata:
 data:
   values.yaml: |
     jaeger:
+      priorityClassName: dkp-high-priority
```

Comment:
I couldn't find any info on this, does the Jaeger custom resource support this?

it looks like there's also `priorityClassName` that we can set on the operator:

https://github.com/jaegertracing/helm-charts/blob/main/charts/jaeger-operator/values.yaml#L96